src/sql/src/session/vars.rs

```diff
@@ -813,19 +813,19 @@ static OPENTELEMETRY_FILTER: Lazy<ServerVar<CloneableEnvFilter>> = Lazy::new(||
 });
 
 /// Controls the connect_timeout setting when connecting to PG via `mz_postgres_util`.
-const PG_SOURCE_CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
+const PG_SOURCE_CONNECT_TIMEOUT: ServerVar<Option<Duration>> = ServerVar {
     name: UncasedStr::new("pg_source_connect_timeout"),
-    value: &mz_postgres_util::DEFAULT_CONNECT_TIMEOUT,
+    value: &None,
```

Comment:
When a user runs `SHOW ALL`, what would appear with this change? Will they see the default values that will be set if unconfigured here?

Reply:
Oh good catch here--made these non-optional again and will show their defaults.